cfgov/unprocessed/apps/find-a-housing-counselor/webpack-config.js

```diff
@@ -4,14 +4,14 @@
 
 const BROWSER_LIST = require( '../../../../config/browser-list-config' );
 const webpack = require( 'webpack' );
-const UglifyWebpackPlugin = require( 'uglifyjs-webpack-plugin' );
+const TerserPlugin = require( 'terser-webpack-plugin' );
 
 /* Set warnings to true to show linter-style warnings.
    Set mangle to false and beautify to true to debug the output code. */
-const COMMON_UGLIFY_CONFIG = new UglifyWebpackPlugin( {
+const COMMON_UGLIFY_CONFIG = new TerserPlugin( {
```

Comment:
With the uglify plugin gone maybe change this to `COMMON_MINIFICATION_CONFIG`.

Reply:
Hmmmmmmmmmmmmmmmmmmmmm good point

Reply:
It is done in https://github.com/cfpb/cfgov-refresh/pull/4987/commits/25d9f83e5f2ddd24318ccfbcb69a666f46982308
Also, please sign off on https://github.com/cfpb/capital-framework/pull/917